webapp/src/Controller/Jury/JudgehostController.php

```diff
@@ -98,10 +99,10 @@ public function indexAction(Request $request): Response
 
         $now           = Utils::now();
         $contest       = $this->dj->getCurrentContest();
-        $query         = 'SELECT judgehost, SUM(IF(endtime, endtime, :now) - GREATEST(:from, starttime)) AS `load`
-                          FROM judging
+        $query         = 'SELECT hostname, SUM(IF(endtime, endtime, :now) - GREATEST(:from, starttime)) AS `load`
+                          FROM judging INNER JOIN judgehost on judging.judgehostid = judgehost.judgehostid
```

Comment:
This could be avoided by just keying the workloads in the `$result` array and below by the judgehostid.